src/App.js

```diff
@@ -1,49 +1,40 @@
-import React, { Component } from 'react';
-import axios from 'axios';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import axios from "axios";
+import "./App.css";
 // import CardList from './CardList';
-import fetchCards from './Scryfall'
-import CubeListContainer from './CubeList';
-import {BrowserRouter, Route} from 'react-router-dom';
+import CubeListContainer from "./CubeList";
+import { BrowserRouter, Route } from "react-router-dom";
 
-const CardListItem = props => (
-    <img src={props.img} alt="" height="350"/>
-);
+const CardListItem = props => <img src={props.img} alt="" height="350" />;
 
 const CardList = props =>
   props.items.map(item => <CardListItem key={item.name} {...item} />);
 
-class CubeButton extends React.Component {
+// class CubeButton extends React.Component {
+//   render() {
+//     const { variant, content, ...others } = this.props;
 
-  render() {
-    const { 
-      variant,
-      content,
-      ...others
-    } = this.props;
-    
-    return (
-      <button className={variant} {...others}>
-        {content}
-      </button>
-    )
-  }
-}
+//     return (
+//       <button className={variant} {...others}>
+//         {content}
+//       </button>
+//     );
+//   }
+// }
```

Comment:
:O Does it comment out any code it sees isn't used?

Reply:
> I take it you tested it? It all looks fine. Though I think we now know it was definitely me doing 4 space indentation - I blame python.

I'm pretty sure vim wanted me to use 4 spaces too, so you're not alone.

> :O Does it comment out any code it sees isn't used?

Not sure what happened there. I'll fix it before merging.

Reply:
Oh, I see what happened (this split confused me). Yeah, I manually commented this out to fix a linting error.